.github/workflows/pypi-upload.yaml

```diff
@@ -0,0 +1,32 @@
+on:
+  workflow_dispatch:
```

Comment:
Note that I added this so that we can manually push to PyPI without having to cut a new release.